i18n/fr.yaml

```diff
@@ -5,74 +5,62 @@ twitter_description:
+  other: "L'application mobile fonctionne en arrière-plan et n'exige aucune interaction avec l'utilisateur après l'intégration. Elle utilise le Bluetooth pour recueillir et partager des identifiants aléatoires avec les téléphones situés à proximité, sur lesquels COVID Shield est installé. Si un utilisateur est testé positif au COVID-19, il peut choisir de partager anonymement ses données de manière que d'autres puissent être avisés d'une possible exposition. Le partage des identifiants aléatoires se fait sur une base volontaire et n'est possible que si le résultat positif du test a été confirmé par un professionnel de santé. COVID Shield télécharge périodiquement les identifiants aléatoires partagés depuis le serveur et les compare sur l'appareil de chaque utilisateur pour déterminer si une exposition possible a eu lieu."
 server_title:
-  other: "Server"
+  other: "Serveur"
 server_content:
-  other: "The server securely collects and stores random IDs uploaded from the mobile app after a confirmed positive test result. It also generates unique temporary codes which grant permission for people with a positive COVID-19 test result to upload their random IDs. The server is flexible enough to be deployed at various levels of government."
+  other: "Le serveur recueille et stocke de façon sécurisée les identifiants aléatoires importés depuis l'application mobile suite à un résultat de test positif confirmé. Il génère également des codes temporaires uniques qui accordent aux individus testés positifs au COVID-19 l'autorisation de mettre en ligne leurs identifiants aléatoires. Le serveur est suffisamment flexible pour un déploiement à différents ordres de gouvernement."
 web_portal_title:
-  other: "Web portal"
+  other: "Portail web"
 web_portal_content_1:
-  other: "The web-based results portal provides health care professionals unique temporary codes which can be shared with users who have tested positive for COVID-19. This code gives a user access to upload their random IDs via the mobile app if they choose. No personal information is collected and there is no association between the codes and specific tests."
+  other: "Le portail web de résultats fournit aux professionnels de santé des codes temporaires uniques qui peuvent être partagés avec les utilisateurs ayant été testés positifs au COVID-19. Ces codes donnent à l'utilisateur un accès lui permettant de mettre en ligne ses identifiants aléatoires via l'application mobile, s'il le souhaite. Aucun renseignement personnel n'est recueilli et il n'existe aucune association entre les codes et des tests particuliers."
```

Comment:
1- "testé positif" is an anglicism, we use "reçoit un résultat de test positif"
2- we say "à la" COVID-19, and not "au" COVID-19
So this paragraph becomes:
`Le portail web de résultats fournit aux professionnels de santé des codes temporaires uniques qui peuvent être partagés avec les utilisateurs ayant reçu un résultat de test positif à la COVID-19. Ces codes donnent à l'utilisateur un accès lui permettant de mettre en ligne ses identifiants aléatoires via l'application mobile, s'il le souhaite. Aucun renseignement personnel n'est recueilli et il n'existe aucune association entre les codes et des tests particuliers.`

Reply:
My suggestions @annemarieleger  : 
• portail Web (with a capital `W`)
• professionnels de `la` santé